requirements_test_all.txt

```diff
@@ -754,7 +754,7 @@ pyatag==0.3.5.3
 pyatmo==5.2.3
 
 # homeassistant.components.apple_tv
-pyatv==0.8.1
```

Comment:
manifest.json needs to be updated.

Reply:
Fixed and tested, working as expected now.